config/offices.yml

```diff
@@ -45,19 +45,25 @@ defaults: &defaults
 development:
   <<: *defaults
   offices:
-    staging_sfax: # Technically this is to our sfax production fax; cuz you
-                  # can't fax urself
+    staging_sfax: # Technically this is to our sfax production fax;
+                  # because you can't fax yourself
       fax_number: "+18448487565"
+      email: "staging@example.com"
     union:
       fax_number: "+16173963015"
+      email: "union@example.com"
     clio:
       fax_number: "+16173963015"
+      email: "clio@example.com"
 production:
   <<: *defaults
   offices:
     staging_sfax:
       fax_number: "+18888433549"
+      email: "gcfengineering+heroku@codeforamerica.org"
```

Comment:
I don't have access to this inbox -- should we use it anyway?

Reply:
@jessieay is (maybe?) the Front mailbox the best address for this right now?

Maybe we should make a temporary @cylinder.work address or alias that this sort of stuff gets funneled through?

Reply:
oh yeah, front might be good! we'd just want to make it clear why something was coming from staging into front